### DISCUSSION TITLE
My angular SSR aplication dont work fine when i disable JS

### DISCUSSION THREAD

**Discussion by giovannidias1** [ngrx/platform#3503] [Q&A] [answered]
When I disable JS in my browser to test the page generated via server, only what is static like header and footer is shown, what comes from Effects to be saved in the Store is not shown.

I tried using ngrx-universal-rehydrate, as a way of trying to work around it 

i'm lost i don't know what to do someone help me

**Comment by brandonroberts** [comment]
You may want to file an issue on that repo. The library is maintained by @yharaskrik

**Reply by yharaskrik** [reply]
Feel free to file an issue. It's not something I tested for though. 

I am away on vacation for the next 3 days just FYI.

**Reply by giovannidias1** [reply]
> You may want to file an issue on that repo. The library is maintained by @yharaskrik

ngrx-universal-rehydrate came after I ran into this problem, as a way of trying to work around it.

**Reply by yharaskrik** [reply]
Are you sure this is an NgRx issue then? Not sure why ngrx would affect the server side rendered page if JS is disabled on the browser.

**Reply by giovannidias1** [reply]
> Are you sure this is an NgRx issue then? Not sure why ngrx would affect the server side rendered page if JS is disabled on the browser.

JS is not influencing it was just the way I found to test server render. The problem is what's in the store is not rendered on the server side. 

(I don't know how to express myself very well in English, sorry)

**Comment by giovannidias1** [comment, marked as answer]
I found the problem the problem is with ngx-editor  a text editor lib, which uses ProseMirror, and with JS disabled ngx-editor doesn't work and breaks the page now I just need to find a way to run it after generating the page on the server.

I even found an issue from @yharaskrik
